src/api/index.ts

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react';
+import axios, { AxiosResponse } from 'axios';
+
+enum RequestMethods {
+  Get = 'GET',
+  Post = 'POST',
+  Put = 'PUT',
+  Delete = 'DELETE'
+}
+
+interface RequestParams {
+  method: RequestMethods,
+  uri: string,
+  data?: any,
+  config?: any
+}
```

Comment:
제네릭 타입을 사용하면 더 단단한 타입이 될 것 같아요! 그리고 config는 `axios`에서 사용하는 타입이 있을 것 같은데, 그걸 가져와서 사용해보는 것은 어떨까유 👀

```suggestion
interface RequestParams<T> {
  method: RequestMethods,
  uri: string,
  data?: T,
  config?: any
}
```

Reply:
애니스크립트를 막아야합..읍..😵